client/src/views/header/ProfileHeaderHomepagePopover.vue

```diff
@@ -120,11 +139,23 @@ async function onOptionClick(option: ProfilePopoverHomepageOption): Promise<void
   });
 }
 
+async function hideParsecDownload(): Promise<void> {
+  const config = await storageManager.retrieveConfig();
+  await storageManager.updateConfig({ hideParsecDownload: true });
+  hideParsecDownloadContent.value = config.hideParsecDownload;
+  await storageManager.storeConfig(config);
+}
```

Comment:
Check the code for `updateConfig`, it already calls `retrieveConfig()` then `storeConfig()`. So all these calls are redundant.

Just update the value and then call `updateConfig()`: 
```
async function hideParsecDownload(): Promise<void> {
  hideParsecDownloadContent.value = true;
  await storageManager.updateConfig({ hideParsecDownload: true });
}
```